Cargo.toml

```diff
@@ -55,7 +55,7 @@ features = [
 
 [target.'cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "openbsd", target_os = "netbsd"))'.dependencies]
 wayland-client = { version = "0.20.10", features = [ "dlopen", "egl", "cursor"] }
-smithay-client-toolkit = "0.2.6"
+smithay-client-toolkit = { git = "https://github.com/Smithay/client-toolkit" }
```

Comment:
You can use 0.3.0 now.

Reply:
I'm going to change this pull request to 'Upgrade to smithay-client-toolkit 0.3.0' as 0.3.0 contains few breaking changes from https://github.com/Smithay/client-toolkit/pull/25

Reply:
Sorry my bad, it seems none of those breaking changes apply